doc/rados/configuration/ceph-conf.rst

```diff
@@ -408,87 +418,83 @@ Every configuration option is typed with one of the types below:
 
-* ``ceph config get <who> <option>`` will show a configuration value
-  stored in the monitors' configuration database for a specific daemon
-  or client (e.g., ``mds.a``), or, if not present in the monitors'
+* ``ceph config get <who> <option>`` shows either a configuration value
+  stored in the monitor configuration database for a specific daemon or client
+  (for example, ``mds.a``), or, if that value is not present in the monitor
   configuration database, the compiled-in default value.
 
-* ``ceph config set <who> <option> <value>`` will set a configuration
-  option in the monitors' configuration database.
+* ``ceph config set <who> <option> <value>`` specifies a configuration
+  option in the monitor configuration database.
 
-* ``ceph config show <who>`` will show the reported running
-  configuration for a running daemon.  These settings may differ from
-  those stored by the monitors if there are also local configuration
-  files in use or options have been overridden on the command line or
-  at run time.  The source of the option values is reported as part
-  of the output.
+* ``ceph config show <who>`` shows the configuration for a running daemon. 
+  These settings might differ from those stored by the monitors if there are
+  also local configuration files in use or if options have been overridden on
+  the command line or at run time. The source of the values of the options is
+  displayed in the output.
 
-* ``ceph config assimilate-conf -i <input file> -o <output file>``
-  will ingest a configuration file from *input file* and move any
-  valid options into the monitors' configuration database.  Any
-  settings that are unrecognized, invalid, or cannot be controlled by
-  the monitor will be returned in an abbreviated config file stored in
-  *output file*.  This command is useful for transitioning from legacy
-  configuration files to centralized monitor-based configuration.
+* ``ceph config assimilate-conf -i <input file> -o <output file>`` ingests a 
+  configuration file from *input file* and moves any valid options into the
+  monitor configuration database. Any settings that are unrecognized, are
+  invalid, or cannot be controlled by the monitor will be returned in an
+  abbreviated configuration file stored in *output file*. This command is
+  useful for transitioning from legacy configuration files to centralized
+  monitor-based configuration.
 
 Note that ``ceph config set <who> <option> <value>`` and ``ceph config get
-<who> <option>`` aren't symmetric because the latter also shows compiled-in
-default values.  In order to determine whether a configuration option is
-present in the monitors' configuration database, use ``ceph config dump``.
-
+<who> <option>`` will not necessarily return the same values. The latter
+command will show compiled-in default values. In order to determine whether a
+configuration option is present in the monitor configuration database, run
+``ceph config dump``.
```

Comment:
As a noob with some experience using these commands and non-native English language, or maybe I'm just thick like that, but I struggled to understand what exactly this means:
* The former command *sets* a value. 
* The latter command *gets* a value.

Testing this in Quincy it seems that `ceph config get` will display the compiled-in default values *if the value has not been modified by user configuration*. I assume this part is about that case? Unless changed in later versions perhaps this could be explained somehow instead of just leaving it at "sometimes they don't match" (paraphrasing the "not necessarily").

Reply:
@bluikko: First of all, on the evidence of your contributions here in the Conversation tab of this PR, I would never have guessed that you were anything but a distinguished-level native speaker of English. So know that whatever it feels like in there, from out here you look to a skilled native speaker of English like a peer, producing distinguished-level English. 

Second of all, I'm going to improve this section in a separate PR that I will raise in the coming week. You will be added as a reviewer of that PR, and we won't merge it until you have been consulted and satisfied. Thanks for letting me know that this part is obscure.